package.json

```diff
@@ -13,7 +13,31 @@
   ],
   "scripts": {
     "test": "tape test.js | tap-spec && standard",
-    "ci": "tape test.js && node testRandom && standard"
+    "ci": "tape test.js && node testRandom && standard",
+    "release": "read -p 'GITHUB_TOKEN: ' GITHUB_TOKEN && export GITHUB_TOKEN=$GITHUB_TOKEN && release-it"
+  },
+  "release-it": {
+    "github": {
+      "release": true
+    },
+    "git": {
+      "tagName": "v${version}",
+      "commitMessage": "chore(release): ${version}"
+    },
+    "hooks": {
+      "before:init": [
+        "npm run test"
+      ]
+    },
+    "npm": {
+      "publish": true
+    },
+    "plugins": {
+      "@release-it/conventional-changelog": {
+        "preset": "angular",
+        "infile": "CHANGELOG.md"
+      }
+    }
```

Comment:
Sure Matteo, just wondering if you would like me to help you manage this repo we can leave it there or i can simply teach you how that works.

Essentially you run: `npm run release` that will start an interactive cli that firstly ask you if you want a patch/minor/major bump and then propts you if you want to push changes, create a release and release to npm, you just have to follow the steps (first prompt is to set GH token env var to create the release to github, without that it will only create a draft release)

Reply:
This kind of complexity is really something I don't want to be dealing with.